recipe/meta.yaml

```diff
@@ -1,23 +1,25 @@
 {% set name = "GLM" %}
-{% set version = "0.9.9.8" %}
+{% set version = "1.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/g-truc/glm/releases/download/{{ version }}/glm-{{ version }}.zip
-  sha256: 37e2a3d62ea3322e43593c34bae29f57e3e251ea89f4067506c94043769ade4c
+  url: https://github.com/g-truc/glm/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 9f3174561fd26904b23f0db5e560971cbf9b3cbda0b280f04d5c379d03bf234c
+  
   patches:
-    - patches/macos_test_errors.patch
-    - patches/cmake_install_target.patch
+     - patches/macos_test_errors.patch
+  #   - patches/cmake_install_target.patch
```

Comment:
Can you just delete this line (and the related file) instead of commenting? Thanks!